### DISCUSSION TITLE
Getting an error on the sample Move code

### DISCUSSION THREAD

**Discussion by icefloe01** [pybricks/support#586] [Q&A] [answered]
The code with two motors. I copied and pasted, hit the "play" button and get an error about an unexpected indent on line 19. I'm still trying to do this with my phone because I'm impatient and don't want to wait until Tuesday for the Bluetooth adaptor to arrive for my laptop. 😂
![Screenshot_20220108-133131_Chrome](https://user-images.githubusercontent.com/3160766/148655769-1686973f-347d-464d-a142-0912ed3c8b3e.jpg)

**Comment by dlech** [comment]
Close the keyboard so you can see the code. In Python, indentation matters, so something probably didn't copy/paste right.

**Comment by laurensvalk** [comment, marked as answer]
Can you take another screenshot of your code, but with the keyboard closed?

**Reply by icefloe01** [reply]
Okay, got it. It seems to look correct by comparison.
![Screenshot_20220108-141838_Chrome](https://user-images.githubusercontent.com/3160766/148656917-8dc7a151-136e-44f1-82a8-6ccfa3ebde2b.jpg)

**Reply by icefloe01** [reply]
I don't think I'm going to be able to get all the necessary keyboard functions with a mobile device unless I pair an actual keyboard to it.

**Reply by dlech** [reply]
Looks like it added an extra indentation on every single line after `while True:` for some reason.

**Reply by icefloe01** [reply]
Okay, got that taken care of and it loads now. I've got a lot of reading and learning to do.

**Reply by icefloe01** [reply]
Thank you!

**Reply by icefloe01** [reply]
One other question about this sample code: am I supposed to be able to manually turn motor A to control motor B?

**Reply by laurensvalk** [reply]
That’s what this particular example does, yes.

So you can probably already use this for your roller coaster 😄

Just change the ports to C and/or D.

**Reply by icefloe01** [reply]
Hmm, another error may have gotten into it then. When I hit the play button, motor "A" seems to set itself to zero, but when I turn it after that, motor "B" does nothing.

**Reply by laurensvalk** [reply]
If you can, please include a video and a copy or screenshot of the code you are using.

**Reply by icefloe01** [reply]
I'll wait until I get the single for my laptop. It has now also stopped setting motor A to zero. If I change it to "C" it does set the motor plugged into C to position zero but the motor plugged into C still work not act as a control. I'm not terribly concerned with it for now as I really still don't know what I'm doing and I trying to do it from a mobile phone browser interface is rather cumbersome. Once I start working from the laptop, if I need to I'll just start a new thread.

Thank you for everything!

**Reply by icefloe01** [reply]
Well it turns out I can't do it on the laptop after all. It's running Windows 8.1 and web Bluetooth requires 10. This thing has already failed an upgrade to 11 and I really don't feel up to the frustration if going through all that with an attempt at 10. Plus it's a work laptop so there's only so much I can do. Off to find a different device to program with.
